docs/components/modeler/bpmn/multi-instance/multi-instance.md

```diff
@@ -94,13 +94,14 @@ target: output
 
 A `completionCondition` defines whether the multi-instance body can be completed immediately when the condition is satisfied. It is a [boolean expression](/components/concepts/expressions.md#boolean-expressions) that will be evaluated each time the instance of the multi-instance body completes. Any instances that are still active are terminated and the multi-instance body is completed when the expression evaluates to `true`.
 
-The BPMN 2.0 specification defines the following properties of a multi-instance body instance:
+The BPMN 2.0 specification defines the following properties of a multi-instance body:
 
 - `numberOfInstances`: The number of instances created.
 - `numberOfActiveInstances`: The number of instances currently active.
 - `numberOfCompletedInstances`: The number of instances already completed.
+- `numberOfTerminatedInstances`: The number of instances already terminated, this will always be `0`.
 
-These properties are available for use in the `completionCondition` expression. Although they are available in this expression, they do not exist as process variables. These properties take precedence over process variables with the same name.
+These properties are available for use in the `completionCondition` expression (eg. when 50% of the instances have completed, the multi-instance body is completed). Although they are available in this expression, they do not exist as process variables. These properties take precedence over process variables with the same name.
```

Comment:
🔧 I'd like to suggest the following changes, focussing on an example expression along with the use case example.

 ```suggestion
These properties are available for use in the `completionCondition` expression. For example, using these properties you can express "complete the multi-instance body when 50% or more of the instances already completed" as `= numberOfCompletedInstances / numberOfInstances >= 0.5`. Although these properties are available in this expression, they do not exist as process variables. These properties take precedence over process variables with the same name.
```